lib/core/src/persist/migrations.rs

```diff
@@ -183,5 +183,26 @@ pub(crate) fn current_migrations() -> Vec<&'static str> {
 
         DROP TABLE old_chain_swaps;
         ",
+        "CREATE TABLE IF NOT EXISTS sync_state(
+            data_id TEXT NOT NULL PRIMARY KEY,
+            record_id TEXT NOT NULL,
+            record_revision INTEGER NOT NULL
+        ) STRICT;",
+        "CREATE TABLE IF NOT EXISTS sync_settings(
+            id INTEGER NOT NULL PRIMARY KEY,
+            remote_url TEXT NOT NULL,
+            latest_revision INTEGER NOT NULL DEFAULT 0
+        ) STRICT;",
+        "CREATE TABLE IF NOT EXISTS sync_outgoing(
+            record_id TEXT NOT NULL PRIMARY KEY,
+            record_type INTEGER NOT NULL,
+            updated_fields_json TEXT
+        ) STRICT;",
+        "CREATE TABLE IF NOT EXISTS sync_incoming(
+            revision INTEGER NOT NULL PRIMARY KEY,
+            id TEXT NOT NULL,
```

Comment:
record_id ?

Reply:
Should be also unique so we will have only one incoming update for every record (the latest one).